pkg/controller/controller.go

```diff
@@ -440,3 +462,10 @@ func (c *CSIControllerService) ControllerExpandVolume(ctx context.Context, req *
 		NodeExpansionRequired: false,
 	}, nil
 }
+
+func isNeedRawPart(params map[string]string) bool {
```

Comment:
`isNeedRawPart` sounds confusing. You can use `isNeedForRawPart` or just `isRawPart` instead